saleor/graphql/utils.py

```diff
@@ -183,3 +183,9 @@ def create_jwt_payload(user, context=None):
     payload["is_staff"] = user.is_staff
     payload["is_superuser"] = user.is_superuser
     return payload
+
+
+def get_requester_from_context(context):
```

Comment:
Maybe `get_user_or_service_account_from_context`? A bit long but I assume we could also have `get_user_from_context` and just `get_service_account_from_context`.